app/src/main/java/com/gh4a/db/SuggestionsProvider.java

```diff
@@ -12,11 +12,14 @@
 import android.support.annotation.NonNull;
 import android.util.Log;
 
+import com.gh4a.BuildConfig;
+
 public class SuggestionsProvider extends ContentProvider {
     private static final String TAG = "SuggestionsProvider";
 
     public interface Columns extends BaseColumns {
-        Uri CONTENT_URI = Uri.parse("content://com.gh4a.SuggestionsProvider/suggestions");
+        Uri CONTENT_URI = Uri.parse(String.format("content://%s.SuggestionsProvider/suggestions",
+                                                  BuildConfig.APPLICATION_ID));
```

Comment:
Code style: As far as I remember we use 8 spaces as continuation indent instead of aligning with first parameter.

For me personally code aligned like you've done it here is harder to read because of less available space.  

This applies to other places too.